core/src/test/resources/org/apache/calcite/test/SqlToRelConverterTest.xml

```diff
@@ -4179,7 +4465,7 @@ when matched and target.sal > 10 then
 when matched and target.sal < 10 then
   update set sal = target.sal + source.sal + 10
 when matched and target.sal = 10 then
-  update set sal = -10
+  DELETE
```

Comment:
There's a few changes to SQL strings, which occurred when I copied the "actual" generated XML file into this one.